python/lsst/afw/display/interface.py

```diff
@@ -855,15 +845,16 @@ def interact(self):
                         "Display._callbacks['{0}']({0},{1},{2}) failed: {3}".format(k, x, y, e))
 
     def setCallback(self, k, func=None, noRaise=False):
-        """Set the callback for a key
+        """Set the callback for a key.
```

Comment:
Add "Backend displays may provide an equivalent graphical UI action, but must make the associated key letter visible in the UI in some way."